app/transformers/business/application.js

```diff
@@ -0,0 +1,29 @@
+export function transformOrganisationApplicationToBusinessApplications (applications) {
+  if (!applications) {
+    return applications
+  }
+
+  const result = []
+  for (const response in applications) {
+    result.push({
+      applicationStatus: {
+        id: response.application_id,
+        open: response.application_code,
+        status: response.status,
+        type: response.application_type_de,
```

Comment:
Presumably mapping provided by James or Ian?

Reply:
I created a Teams chat with Ian, James and you and brought up this topic, please let me know if I could do anything else